src/model/enhance-blockquotes.test.ts

```diff
@@ -10,10 +10,14 @@ describe('Enhancing blockquotes', () => {
 	});
 
 	it('adds the quoted prop when the quoted class was found', () => {
-		const input = {
+		const input: CAPIType = {
```

Comment:
`enhanceBlockquotes` expect param to be of type `CAPIType`, so we need to add type definition. But doing so means we have some missing keys in the object (`id`, `primaryDateLine ` and `secondaryDateLine`)